Playbooks/AS-Block-GitHub-User/README.md

```diff
@@ -0,0 +1,290 @@
+# AS-Block-GitHub-User
+
+Author: Accelerynt
+
+For any technical questions, please contact info@accelerynt.com  
+
+[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FPlaybooks%2FAS-Block-GitHub-User%2Fazuredeploy.json)
+[![Deploy to Azure Gov](https://aka.ms/deploytoazuregovbutton)](https://portal.azure.us/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FPlaybooks%2FAS-Block-GitHub-User%2Fazuredeploy.json)       
+
+This playbook is intended to be run from a Microsoft Sentinel Incident. It will look up the GitHub Users associated with the Incident Account Entities and block them from your GitHub Organization. If they are members of the GitHub Organization, they will also be removed. A comment noting the affected GitHub Users will be added to the Incident.
+
+![BlockGitHubUser_Demo_1](Images/BlockGitHubUser_Demo_1.png)
+
+![BlockGitHubUser_Demo_2](Images/BlockGitHubUser_Demo_2.png)
+ 
+                                                                                                                             
+#
+### Requirements
+                                                                                                                            
+The following items are required under the template settings during deployment: 
+
+* A [GitHub App](https://github.com/Azure/Azure-Sentinel/tree/master/Playbooks/AS-Block-GitHub-User#install-the-github-app) with permissions to read and write on Users in your GitHub Organization
+* The [GitHub App Installation ID](https://github.com/Azure/Azure-Sentinel/tree/master/Playbooks/AS-Block-GitHub-User#install-the-github-app)
+* An [Encoded Private Key](https://github.com/Azure/Azure-Sentinel/tree/master/Playbooks/AS-Block-GitHub-User#encode-the-private-key-for-storage-compatibility-in-azure-key-vault) for your GitHub App
+* An [Azure Key Vault Secret](https://github.com/Azure/Azure-Sentinel/tree/master/Playbooks/AS-Block-GitHub-User#create-an-azure-key-vault-secret) containing your encoded private key
+* Install [Visual Studio Code](https://github.com/Azure/Azure-Sentinel/tree/master/Playbooks/AS-Block-GitHub-User#configure-visual-studio-code) and configure it to deploy an Azure Function to your Azure tenant
+* An [Azure Function App](https://github.com/Azure/Azure-Sentinel/tree/master/Playbooks/AS-Block-GitHub-User#deploy-the-azure-function-app) that supports Node.js to deploy an Azure function to
```

Comment:
These links throw 404 error.

Reply:
@rahul0216 These are links to other sections in the readme document. The URLs won't exist until the PR is merged- this 404 error is expected because the readme file doesn't exist on the master branch yet.